### DISCUSSION TITLE
Hard to find: Settings for "Double Opt-In Email"

### DISCUSSION THREAD

**Discussion by FrankJer** [hizzle-co/noptin#586] [Ideas]
Hi Brian,

it took a while until I found this: Settings --> Emails --> Double Opt-In Email

I think it would be a good idea to somehow highlight the Double Opt-In Email settings or make this part of the section easier to find in any other way.

Cheers, Frank

**Comment by picocodes** [comment]
Thanks, Frank. The upcoming update will address this.
